widget/completionentry.go

```diff
@@ -2,21 +2,26 @@ package widget
 
 import (
 	"fyne.io/fyne/v2"
+	"fyne.io/fyne/v2/driver/desktop"
```

Comment:
Seems like this will be a problem on mobile builds...

Reply:
You link link to the abstraction... just don't assume desktop types if it's mobile runtime.

Reply:
You are right, I removed this, and the auto-item-select on hover.
It doesn't really make sens, it was just a personal flavor